virtool/users/api.py

```diff
@@ -32,7 +32,7 @@
 @routes.view("/users")
 class UsersView(PydanticView):
     @policy(AdministratorRoutePolicy)
-    async def get(self) -> Union[r200[User], r403]:
+    async def get(self, find: Optional[str]) -> Union[r200[User], r403]:
```

Comment:
Use `Field` to add a description so this can be shown in the docs. Write a description similar to that for labels.